build.jenkins

```diff
@@ -1,35 +0,0 @@
-pipeline {
-  agent none
-  stages {
-
-    stage('Build') {
-      agent {
-        dockerfile {
-         reuseNode true
-        }
-      }
-      steps {
-        sh './gradlew clean build'
-      }
-    }
-
-    stage('Tests') {
-      agent {
-        dockerfile {
-          reuseNode true
-        }
-      }
-      steps {
-        sh 'echo "Running unit tests and integration tests"'
-        sh './gradlew clean test intTest'
```

Comment:
> intTest

What is doing this? I can't immediately see anything in the Concourse pipeline that does this step...

Reply:
Let me check the config.

Reply:
Good spot. @jakubmiarka Do you know if this need to be running the intTests in concourse - https://github.com/alphagov/verify-terraform/blob/master/environments/tools/platform/files/pipelines/verify-service-provider.yml#L814

Reply:
possibly? ... The VSP pipeline is for VSP and we pull the LMS just to use it (to support the e2e journey)... So we should run the intTests somewhere...

Reply:
Got you. We will migrate this to build-libraries on concourse then to build and run the tests.

Reply:
Ok. We're running these now in Travis.